frontend/android/app/src/main/kotlin/app/ehrenamtskarte/MainActivity.kt

```diff
@@ -1,6 +1,49 @@
 package app.ehrenamtskarte
 
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.preference.PreferenceManager
+import android.util.Log
 import io.flutter.embedding.android.FlutterActivity
+import java.io.File
 
 class MainActivity: FlutterActivity() {
+
+    private fun resetApp() {
+        val m = packageManager
+        try {
+            val info = m.getPackageInfo(packageName, 0)
+            deleteRecursive(File(info.applicationInfo.dataDir))
+        } catch (e: PackageManager.NameNotFoundException) {
+            Log.e("WRAPPER", "Failed to get data directory!", e)
+        }
```

Comment:
Do we also need to reset the shared preferences? Or are they stored within this directory?

Reply:
I thin shared preferences are in the dataDir. So not need to delete them.